### ISSUE TITLE
[Bug]  Tips icon missing 

### ISSUE THREAD

**Issue by uldtot** [IceWhaleTech/CasaOS#1623]
Missing icon for Tips on Apps.
Expected an icon to click on. The icon is there, just not shown. 

![image](https://github.com/IceWhaleTech/CasaOS/assets/3944295/45e4a11e-be65-4508-80d9-72ad771332ec)

```
 - CasaOS v0.4.5 on a Zimaboard (Acquired December 2023 from Icewhale, Allen knows more details if it's needed)
```

**System Time**

> Run `timedatectl` and share the output

```
               Local time: Thu 2024-01-18 16:06:54 EST
           Universal time: Thu 2024-01-18 21:06:54 UTC
                 RTC time: Thu 2024-01-18 21:06:54
                Time zone: America/New_York (EST, -0500)
System clock synchronized: yes
              NTP service: active
          RTC in local TZ: no
```

**Comment by jerrykuku** [comment]
we will fix this soon.

**Comment by LinkLeong** [comment]
We have released 0.4.8 with a fix for this issue, thanks for your feedback!
